invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js

```diff
@@ -121,6 +140,10 @@ export const RecordVersionsList = ({ record, isPreview }) => {
         </Placeholder>
       </>
     )
+  ) : error ? (
+    <div className="rel-mr-1 rel-ml-1">
```

Comment:
the div is not needed, you can add the classes on the message directly if I am not mistaken